pallets/asset/src/tests.rs

```diff
@@ -226,6 +227,110 @@ fn asset_issue_should_succeed() {
 	});
 }
 
+#[test]
+fn asset_issue_should_succeed_until_limit() {
```

Comment:
nit: Always name the test what it does. 
`asset_issue_should_fail_when_distribution_limit_exceeds`.

Reply:
Resolved this.